src/main/flight/gyroanalyse.c

```diff
@@ -112,13 +152,15 @@ void gyroDataAnalyseInit(uint32_t targetLooptimeUs)
     // otherwise we need to calculate a FFT sample frequency to ensure we get 3 samples (gyro loops < 4K)
     const int gyroLoopRateHz = lrintf((1.0f / targetLooptimeUs) * 1e6f);
     
-    fftSamplingRateHz = MIN((gyroLoopRateHz / 3), fftSamplingRateHz);
+    fftSamplingRateHz = MIN((gyroLoopRateHz / 3), fftSamplingRateHz); // 1333hz at 8k medium
+
+    fftResolution = (float)fftSamplingRateHz / FFT_WINDOW_SIZE; // 41.65hz per bin for medium
 
-    fftResolution = (float)fftSamplingRateHz / FFT_WINDOW_SIZE;
+    fftStartBin = MAX(2, dynNotchMinHz / lrintf(fftResolution)); // can't use bin 0
 
-    fftStartBin = dynNotchMinHz / lrintf(fftResolution);
+    dynNotchMaxCtrHz = fftSamplingRateHz * 0.48; // Stay below Nyquist; frequency at which each axis updates
 
-    dynNotchMaxCtrHz = fftSamplingRateHz / 2; //Nyquist
+    smoothFactor = 2 * 3.14f * DYN_NOTCH_SMOOTH_HZ / (gyroLoopRateHz / 12); // minimum PT1 k value
```

Comment:
Is `3.14` meant to be the value of Pi? If so, you can use `M_PIf` from `maths.h`

Reply:
Thanks, it is.  It doesn't require much accuracy.  What would be computationally most efficient?

Reply:
Computationally they'd be the same, however semantically would be more correct nonetheless.